app/config.py

```diff
@@ -383,6 +383,11 @@ class Config:
                 "schedule": crontab(hour=19, minute=00),
                 "options": {"queue": QueueNames.PERIODIC},
             },
+            "update-report-status-to-deleted": {
```

Comment:
This is a really minor thing, but can we move this in the list so that it's next to `save-daily-notification-processing-time`? The nightly tasks are roughly listed in time order, which makes it easy to tell what's running at what time of night